status-im/status-react
# Moved to new react-native-webview community project

### Summary:

fixes #6398
fixes #7471
relates to #6861
relates to #6300

Migrated browser to latest [react-native-webview](https://github.com/react-native-community/react-native-webview/)

This migration modifies significantly how the browser works on both iOS and android.

## Google safe browsing

Some relevant links:

* https://transparencyreport.google.com/safe-browsing/search?url=malware.testing.google.test%2F&hl=en
* https://developers.google.com/safe-browsing/

## Left to do

* restore ServiceWorker injection support

## Tests

* make sure the testing DApp is augmented to cover more scenario
* add missing tests from the [existing](https://docs.google.com/spreadsheets/d/1xTz1MSDUoeRoNlbd3BrxXu3RzTFwr4FatN2YdxVdn3Y/edit#gid=2060193276) manual tests

## Missing tests

* cookies/localStorage persistence
* file uploads
* file browsing (pdf, .)

status: wip  (only Android migration yet)
